tools/compile_tf_graph.py

```diff
@@ -633,11 +644,6 @@ def cond(i, net_vars, acc_tas, seq_len_info=None)
     # Note that this would be wrong in beam search.
     self.net.get_global_batch_info().dim = rec_layer.get_batch_dim_from_state()
 
-    with tf.name_scope("cond"):
-      rec_layer.create_state_var("cond", tf.constant(True))
-      res = cond(i, net_vars, acc_tas, seq_len_info)
-      rec_layer.set_state_var_final_value("cond", res)
-
```

Comment:
You talked about this. But what exactly was the problem? Can you post some details about the error you get because of this?

Also, just removing this is not really an option anyway. We could maybe make it optional (some option like `add_cond` or so). But before we do that, we should understand what the real problem is. Because I would not expect that this should cause any problem. It should not be a problem to just have this always in the graph, such that it can be used when needed/wanted.